lib/rest.js

```diff
@@ -203,6 +203,10 @@ module.exports = class Rest {
                 //only retry once on refresh since there should be no reason for this token to be invalid
                 return this._apiRequest(requestOptions, 1);
             } else {
+                if (ex?.response?.data?.message) {
+                    ex.message = ex.response.data.message;
+                    ex.code = ex.response.data.errorcode;
+                }
```

Comment:
to be consistent with SOAP it might be good to consider adding RESTError as a custom type?